### ISSUE TITLE
Multiprocessing with num_workers>1 fails with pickle errors

### ISSUE THREAD

**Issue by hagenw** [audeering/opensmile-python#104]
When running the tests in the current main we get:

```python
$ python -m pytest -x
...
TypeError: cannot pickle 'mappingproxy' object
```

If I inspect the problem it uses `multiprocessing=True` and `num_workers=5`.
The error does not appear if I use `multiprocessing=False` or `num_workers=1`.

@frankenjoe seems like they changed something in Python?

**Comment by frankenjoe** [comment]
I guess though. Bu you have not upgraded the Python version, so why does the error suddenly appear?

**Comment by hagenw** [comment]
That is a very good question. I repeated the test locally with Python 3.8 and it fails with the same error message. So I guess, it must be some interaction with another package.

**Comment by frankenjoe** [comment]
Maybe try to downgrade `audinterface` and/or `audobject` as a start?

**Comment by hagenw** [comment]
Turns out `multiprocessing=True` was also broken for earlier versions of `audinterface` (just the error message was differrent), compare https://github.com/audeering/audinterface/pull/163

**Comment by hagenw** [comment]
This is fixed with release 1.2.1 of `audinterface`
